testdata/numbers.txt

```diff
@@ -0,0 +1,48 @@
+# Show single-letter misspelling is ignored when requested.
+
+! gospel -ignore-numbers=false -show=false
+stdout 'main.go:7:1: "1e43" is misspelled in comment'
+stdout 'main.go:7:1: "0xE596B7B0_C643C719" is misspelled in comment'
+stdout 'main.go:7:1: "0xE596B7B0_C643C719_6D9CCD05_D0000000" is misspelled in comment'
+stdout 'main.go:19:44: "1e-348" is misspelled in comment'
+stdout 'main.go:20:44: "1e-347" is misspelled in comment'
+stdout 'main.go:21:44: "1e-346" is misspelled in comment'
+stdout 'main.go:22:44: "1e-345" is misspelled in comment'
+stdout 'main.go:23:44: "1e-344" is misspelled in comment'
```

Comment:
A couple more that come to mind: the hexadecimal `\xa3` as in Go strings, and human-friendly "thousands" quantities like `10k` or `300k`